src/System.Private.CoreLib/shared/System/Globalization/CultureInfo.cs

```diff
@@ -588,7 +589,7 @@ public virtual CultureInfo Parent
             {
                 if (_parent == null)
                 {
-                    CultureInfo culture = null;
+                    CultureInfo? culture = null;
```

Comment:
GitHub won't let me comment on it, but the annotations for the get/set properties above, e.g. DefaultThreadCurrentCulture and DefaultThreadCurrentUICulture don't look correct... you can set those to null, but they're currently annotated as non-nullable.

Reply:
you're right - I initially thought that will produce NullRefException but seems it actually means default